### PR TITLE
[Java] RayCli submit function fixing

### PR DESCRIPTION
**Pull request by lixin-wei**
## What do these changes do?

Fix Java RayCli's submit failure in cluster mode, caused by parameter building mistake.

Set driver output redirection.

## Related issue number

nope.

### CODE REVIEW

**Comment by jovan-wong** [comment]
https://github.com/ray-project/ray/pull/2956/files#diff-58be193099b7f15cc57aa30e2faf44abL142

How about rephrase them with `!isEmpty()`?

**Review by jovany-wang — COMMENTED** [review]
Btw, the `submit()` will be removed from java worker sooner.
see [this](https://github.com/ray-project/ray/pull/2938) for detail.

File: java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java [hunk 1]
```diff
@@ -110,7 +110,7 @@ public Process startDriver(String mainClass, String redisAddress, UniqueId drive
         "",
         ip,
         redisAddress,
-        false,
+        true,
         false,
```
Comment (jovan-wong) [inline java/runtime/src/main/java/org/ray/runtime/runner/RunManager.java]: @raulchen  cc `redirect` argument.

**Review by robertnishihara — APPROVED** [review]
This looks good to me assuming tests pass!

**Comment by raulchen** [comment]
I think this PR is conflicting with #2938. 
@wlx65003 Could you hold on for a while? I'll talk to you.

**Pull request closed without merging**